File: SampleSetup/Protocol/Body/ApiTokenPostArg.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public class ApiTokenPostArg
    {
        [Obsolete("Use the other one")]
        public string UserName = "";

        /// <summary>
        /// A sample summary.
        /// </summary>
        public int UserNameKind = 0;

        /// <summary>
        /// A multiline summary.
        /// test explanations :)
        /// </summary>
        public string Password = "";
    }
}

File: SampleSetup/Output/PHP_8/Body/ApiTokenPostArg.php
<?php
namespace SampleProject\Protocol\Body;

class ApiTokenPostArg {
    /**
     * @deprecated Message: Use the other one
     */
    public string $UserName;

    /**
     * A sample summary.
     */
    public int $UserNameKind;

    /**
     * A multiline summary.
     * test explanations :)
     */
    public string $Password;

    public function __construct()
    {
        $this->UserName = "";
        $this->UserNameKind = 0;
        $this->Password = "";
    }
}
